tools/azure-sdk-tools/packaging_tools/sdk_generator.py

```diff
@@ -76,7 +76,7 @@ def after_multiapi_combiner(sdk_code_path: str, package_name: str, folder_name:
             subfolders_name = [s.name for s in subfolder_path.iterdir() if s.is_dir() and not s.name.startswith("_")]
             content["packaging"]["exclude_folders"] = ",".join([exclude(f"{package_name}-{s}") for s in subfolders_name])
 
-        with open(toml_file, "w") as file_out:
+        with open(toml_file, "wb") as file_out:
```

Comment:
@scbedd Any change for `sdk_generator.py` may influence SDK generation pipeline. So please inform us to test it before change the code. Thanks!

Reply:
@msyyc  You're absolutely right, apologies for the break here.

I'm going to add an integration test for the generator so that I can avoid this in the future.